QEfficient/cloud/infer.py

```diff
@@ -119,6 +134,41 @@ def main(
         full_batch_size=full_batch_size,
     )
 
+    #########
+    # Log
+    #########
+
+    if benchmark:
+        input_len = max([len(x) for x in tokenizer(prompt, return_tensors="np").input_ids])
+        num_chunks = -(input_len // -prompt_len)
+        input_len = num_chunks * prompt_len
+
+        fields = {
+            "MODEL\nNAME": model_name,
+            "BATCH\nSIZE": batch_size,
+            "FULL\nBATCH_SIZE": full_batch_size,
+            "CPL": prompt_len,
+            "PL": input_len,
+            "GL": generation_len,
+            "CL": ctx_len,
+            "CORES": num_cores,
+            "NUM\nSOCS": len(device_group),
```

Comment:
this can be None, you can add
`len(device_group) if device_group else 1`

Reply:
Thanks! Resolved. Please see
https://github.com/quic-morteza/efficient-transformers/blob/8f6abc5a57bd369c4deb1cea8d18725ddc57e2a2/QEfficient/utils/_utils.py#L309